src/routes/data/types.rs

```diff
@@ -102,6 +102,17 @@ pub struct DeleteCardResponse {
     pub status: Status,
 }
 
+#[derive(serde::Deserialize)]
+pub struct FingerprintRequest {
+    pub card_number: storage::types::CardNumber,
```

Comment:
Had a question:
In the current scenerio the input would look something like this:
```json
{
   "card_number": "4242424242424242",
   "hash_key": "abc"
}
```
This is vastly different from that of the store card API, should we have them a bit similar?
Something like:
```json
{
   "card": {
       "card_number": "4242424242424242"
    },
   "hash_key": "abc"
}
```